projects/Four In A Row - Classic/info.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Four In A Row - Classic",
+    "description": "You can now play the classic Four In A Row board game on your phone or tablet! The objective of the game is to connect 4 of your chips in a row. You can do this horizontally, vertically, or diagonal. The first player who does this wins! But watch out when you're playing, as your opponent will try to connect their four chips as well!",
+    "developers": [
```

Comment:
If it is a personal project remove publisher because this is linked to the company name that publishes. (So we can make a difference between personal projects and projects for companies.)

Reply:
Done!